dev/how-to/get-started/data-migration.md

```diff
@@ -106,7 +106,7 @@ $ pgrep -a mysqld
 17782 mysqld --defaults-group-suffix=3
 ```
 
-## Non-overlapping shards
+## Replicate shards
```

Comment:
It's preferable to use a heading pattern such as "verb+noun", so this header is correct. But another header "## Starting DM master and workers" in this document does not use the same pattern. We should update it to unify the usage.

Reply:
OK